runtime/browser/xwalk_browser_context.h

```diff
@@ -147,6 +147,8 @@ class XWalkBrowserContext
   std::unique_ptr<XWalkSSLHostStateDelegate> ssl_host_state_delegate_;
   std::unique_ptr<content::PermissionManager> permission_manager_;
 
+  storage::SpecialStoragePolicy* special_storiage_policy_;
```

Comment:
It looks like this is being leaked; I think you can follow what we do with the pointers above this one and use a `std::unique_ptr` here. I also think you can instantiate it in `GetSpecialStoragePolicy()` itself like the other getters do.